src/app/phase/phase-create/phase-create.component.html

```diff
@@ -1 +1,26 @@
-<button class="btn btn-primary cmbutton" routerLink="/phase">Create Phase</button>
+<form (ngSubmit)="onSubmit()" #form="ngForm">
+  <fieldset>
+    <div class="form-group">
+      <div>
+        <p>phaseNumber</p>
+        <input id="phasenumber" name="phasenumber" class="form-control" required [(ngModel)]="phase.phasenumber"/>
```

Comment:
This field is called `phaseNumber` in the API and, thus, it should be called the same at Phase class of the client side.